store/extensions/PositionGo/extension.json

```
@@ -0,0 +1,33 @@
+{
+    "title": "PositionGo",
+    "description": "Save/Load positions for certain rooms. Move to a position with a single command.",
+    "authors": [
+      {
+        "name": "iUseYahoo",
+        "hotel": ".com",
+        "username": "floppidity",
+        "discord": "Wh1z#6363"
+      }
+    ],
+    "version": "1.0.0",
+    "categories": ["Usability", "Others"],
+    
```

Missing the required stable tag
```suggestion

    "stable": true,
    
```